docs/user_manual/components.md

```diff
@@ -615,6 +615,18 @@ the meaning of `RealValueInput` is different, as shown in the table below.
 | `p_sigma`    | `RealValueInput` | watt (W)                   | standard deviation of the active power measurement error. Usually this is the absolute measurement error range divided by 3.   | &#10060; see the explanation below. | &#10004; | `> 0` |
 | `q_sigma`    | `RealValueInput` | volt-ampere-reactive (var) | standard deviation of the reactive power measurement error. Usually this is the absolute measurement error range divided by 3. | &#10060; see the explanation below. | &#10004; | `> 0` |
 
+Valid combinations of `power_sigma`, `p_sigma` and `q_sigma` are:
+| `power_sigma` | `p_sigma` | `q_sigma` |  result  |
+|:-------------:|:---------:|:---------:|:--------:|
+|    &#10004;   |  &#10004; |  &#10004; |  valid   |
+|    &#10004;   |  &#10004; | &#10060;  | invalid  |
+|    &#10004;   | &#10060;  |  &#10004; | invalid  |
+|    &#10004;   | &#10060;  | &#10060;  |  valid   |
+|   &#10060;    |  &#10004; |  &#10004; |  valid   |
+|   &#10060;    |  &#10004; | &#10060;  | invalid  |
+|   &#10060;    | &#10060;  |  &#10004; | invalid  |
+|   &#10060;    | &#10060;  | &#10060;  | invalid  |
```

Comment:
maybe instead provide just a `x`-mark for "provided" and then a &#10004; or a &#10060; for the `valid/invalid` column. i think that looks easier

Reply:
Changes included in the latest commit.